Sources/Shared/Models/VIMLiveQuota.swift

```diff
@@ -39,7 +46,19 @@ public class VIMLiveQuota: VIMModelObject {
     
     /// The `time` field in a `live_quota` response.
     @objc dynamic public private(set) var time: VIMLiveTime?
-    
+
+    /// The `status` field in a `live_quota` response.
+    @objc dynamic public private(set) var status: String?
+
+    /// The status of the live quota in `LiveQuotaStatus` enum.
+    public var liveQuotaStatus: LiveQuotaStatus? {
+        guard let status = self.status else {
+            return nil
+        }
+
+        return LiveQuotaStatus(rawValue: status)
```

Comment:
Since this is a failable initializer and can return nil if the `status` String comes back from the server incorrectly, how would you feel about moving it into the `guard` like this:

```Swift
public var liveQuotaStatus: LiveQuotaStatus? {
    guard
        let stringValue = self.status,
        let status = LiveQuotaStatus(rawValue: stringValue)
    else {
        return nil
    }

    return
}
```